setup/ci/gcb.md

```diff
@@ -20,35 +22,37 @@ You can enable the API with the following `gcloud` command:
 gcloud services enable cloudbuild.googleapis.com
 ```
 
-## Configure Spinnaker to Listen for Google Cloud Build Pub/Sub Notifications
-
 Google Cloud Build sends [Build Notifications](https://cloud.google.com/cloud-build/docs/send-build-notifications)
-when the state of your build changes. You can create a pipeline trigger to invoke a pipeline based
-on the status of your build.
+when the state of your build changes.  Spinnaker subscribes to these pub/sub messages so that in can track the status
+of builds it has initiated in a GCB stage and to trigger pipelines based on build status changes.
 
-1. Create a Subscription object for the `cloud-builds` topic in your project:
+Create a Subscription object for the `cloud-builds` topic in your project:
 
     ```
     PROJECT_ID=
-    SUBSCRIPTION_NAME=gCloudBuilds
+    SUBSCRIPTION_NAME=spinnaker-cloud-build
 
     gcloud pubsub subscriptions create $SUBSCRIPTION_NAME \
       --topic projects/$PROJECT_ID/topics/cloud-builds \
       --project $PROJECT_ID
     ```
+    
+Finally, you will need a service account that has both Cloud Build Editor and Pub/Sub Subscriber permissions, which is
+assumed to be in a file at path `$SERVICE_ACCOUNT_KEY` in the below commands.
 
-2. Configure and deploy Spinnaker via Halyard with your newly created subscription.
+## Configure Spinnaker for Google Cloud Build Support
 
-    ```
-    hal config pubsub google subscription add $SUBSCRIPTION_NAME \
+Use the following Halyard command to create a GCB account, enable the GCB integration, and re-deploy Spinnaker.
+```
+    hal config ci gcb account add $ACCOUNT_NAME \
       --project $PROJECT_ID \
       --subscription-name $SUBSCRIPTION_NAME \
-      --message-format GCB
-
-    hal config pubsub google enable
+      --json-key $SERVICE_ACCOUNT_KEY
 
+    hal config ci gcb enable
+    
     hal deploy apply
-    ```
+```
 
 ## Configure Your Pipeline Trigger
 
```

Comment:
Above, we're on the command line. Now, suddenly, Deck. Which is fine, but it might be a good idea to put a sentence here orienting the user. 
"Configure your pipeline to be triggered by a completed GCB build:"